acceptance/tests/puppet_device_test.rb

```diff
@@ -0,0 +1,63 @@
+test_name "puppet device is able to run and configure a node"
+
+codedir = "#{master.puppet['codedir']}/modules"
+confdir = master.puppet['confdir']
+common_options="--config /tmp/puppet.conf --debug --trace -w0 --server #{master}"
+
+on(master, puppet("module install --target-dir #{codedir} puppetlabs-test_device"))
+on(master, puppet("module install --target-dir #{codedir} puppetlabs-device_manager"))
+on(master, puppet("module install --target-dir #{codedir} puppetlabs-resource_api"))
+
+apply_manifest_on(master, <<MANIFEST, :catch_failures => true)
+# create and configure test user to trigger PUP-9642
+user {
+  'test':
+    ensure => 'present',
+}
+
+# testfile for --apply
+file {
+  '/tmp/spinner.pp':
+    content => "spinner { '999': ensure => 'absent' }"
```

Comment:
Can we spin just once?

```
  Debug: Current State: {:name=>"999", :ensure=>"present", :trigger=>false}
  ...
  spinner { '999': 
    ensure => 'present',
    trigger => false,
    cpu_time => 0,
    wait_time => 0,
  }

```

Reply:
it's only spinning once. the rest is unmanaged output.

Reply:
Not a blocker, but this causes 3 * 999 lines of output, which makes it difficult to wade through. Can it just be `1`?